README.md

```diff
@@ -547,3 +548,8 @@ provider.request(.showProducts) { result in
 }
 
 ```
+
+## SwiftyJSON Model Generator
+Tools to generate SwiftyJSON Models
+- [jsoncafe.com](http://www.jsoncafe.com/)   //online tool
```

Comment:
- [JSON Cafe](http://www.jsoncafe.com/)
- [JSON Export](https://github.com/Ahmed-Ali/JSONExport)

I would prefer this markdown format.